InnaFeature/Pages/Forms.cs

```diff
@@ -0,0 +1,61 @@
+﻿using OpenQA.Selenium;
+
+namespace InnaFeature.Pages
+{
+    internal class Forms : BasePage
+    {
+        public Forms(IWebDriver driver) : base(driver)
+        {
+
+        }
+        public IWebElement Section(string section) =>
```

Comment:
yes, such elements should be moved to base page class to avoid unnecessary duplications

Reply:
Also, please add a new line after each element declaration to improve readability. Same for each file in Pages folder